components/prism-bqn.js

```diff
@@ -0,0 +1,65 @@
+Prism.languages.bqn = {
+	'shebang': {
+		pattern: /^#![ \t]*\/.*/,
+		alias: 'important',
+		greedy: true
+	},
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'string-literal': {
+		pattern: /"(?:[^"]|"")*"/,
+		greedy: true,
+		alias: 'string'
+	},
+	'character-literal': {
+		pattern: /'.'/,
+		greedy: true,
+		alias: 'string'
+	},
+	'function': {
+		pattern: /•[\w¯.∞π]+[\w¯.∞π]*/i
+	},
+	'dot-notation-on-brackets': {
+		pattern: /\{(?=.*\}\.)|\}\./,
+		alias: 'namespace'
+	},
+	'special-name': {
+		pattern: /(?:𝕨|𝕩|𝕗|𝕘|𝕤|𝕣|𝕎|𝕏|𝔽|𝔾|𝕊|_𝕣_|_𝕣)/,
+		alias: 'keyword'
+	},
+	'dot-notation-on-name': {
+		pattern: /[A-Z_][\w¯∞π]*\./i,
+		alias: 'namespace'
+	},
+	'word-number-scientific': {
+		pattern: /\d+(?:\.\d+)?[e|E]¯?\d+/,
+		alias: 'number'
+	},
+	'word-name': {
+		pattern: /[A-Z_][\w¯∞π]*/i,
+		alias: 'symbol'
+	},
+	'word-number': {
+		pattern: /[¯∞π]?(?:\d*\.?\b\d+(?:e[+¯]?\d+)?|¯|∞|π)(?:j¯?(?:(?:\d+(?:\.\d+)?|\.\d+)(?:e[+¯]?\d+)?|¯|∞|π))?/i,
+		alias: 'number'
+	},
+	'null-literal': {
+		pattern: /@/,
+		alias: 'string'
```

Comment:
Isn't this also a character?
```suggestion
		alias: 'char'
```

Reply:
Technically, it is considered a token, I think: https://mlochbaum.github.io/BQN/spec/token.html 

However, it is not listed as a type: https://mlochbaum.github.io/BQN/spec/types.html

So, it should probably be a char as well.